src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraEventSchema.scala

```diff
@@ -81,6 +81,37 @@ object CassandraPopularTopics {
   }
 }
 
+object CassandraConjunctiveTopics {
+  def apply(item: Event): Seq[ConjunctiveTopicAggregate] = {
+    val keywords = item.computedfeatures.keywords
+    val keywordPairs = if (keywords.size == 1) Seq((keywords.head, null)) else keywords.combinations(2).flatMap(combination => Seq(
```

Comment:
Correct, based on other similar references (e.g. `CassandraEventTopicSchema#apply`), this  should be safe.